R/sugar.R

```diff
@@ -66,3 +66,157 @@ opt = function(.key, ...) {
 opts = function(.keys, ...) {
   dictionary_sugar_mget(mlr_optimizers, .keys, ...)
 }
+
+
+#' @title Black-Box Optimization
```

Comment:
We should make the docs here accessible to people that do not have experience with `paradox` and friends.
This might mean a little more detailed documentation.